api/connector/simpleconnector_test.go

```diff
@@ -100,3 +102,29 @@ func (s *simpleConnectorSuite) TestNewSimpleRespectsClientCredentials(c *gc.C) {
 		}
 	}
 }
+
+func (s *simpleConnectorSuite) TestSimpleConnector_Connect(c *gc.C) {
```

Comment:
Drop the underscore.

```suggestion
func (s *simpleConnectorSuite) TestSimpleConnectorConnect(c *gc.C) {
```

Reply:
I generally like it, I often just go generate tests and it defaults to this. Also helps kind of namespace cases. We don't do it in JIMM either really, so probably best to just not do it and keep things consistent. Will resolve.